dll/win32/shell32/CCopyToMenu.cpp

```diff
@@ -271,6 +271,11 @@ CCopyToMenu::QueryContextMenu(HMENU hMenu,
     TRACE("CCopyToMenu::QueryContextMenu(%p, %u, %u, %u, %u)\n",
           hMenu, indexMenu, idCmdFirst, idCmdLast, uFlags);
 
+    if ((uFlags & (CMF_NOVERBS | CMF_VERBSONLY)) != 0) // CORE-16544
+    {
+        return MAKE_HRESULT(SEVERITY_SUCCESS, 0, 0);
+    }
```

Comment:
I would omit the opening and closing curly braces for conditions where the commands do need only a single line.
Here and also at the other 2 places where such braces were added. That style is a giant waste of vertical space in a source file. Those can be added whenever a 2nd command would get added. And no: Developers won't forget that: There is static code analysis, we are not in the 80ies anymore. ;-)